doc/main.html

```diff
@@ -1235,6 +1249,272 @@ <h3>Git commit hooks</h3>
 
   </section>
 
+
+  <section class="annex" id="sec-document-status-workflow">
+    <h2>Document Status Workflow</h2>
+    
+    <section id="sec-document-mapping-status">
+      <h3>Mapping <code>pubState</code> and <code>pubStage</code> to Document Status</h3>
+      <p>
+        <a href="#tab-document-mapping-status"></a> maps the required values of <code>content</code> for <code>pubState</code> and <code>pubStage</code> during the various stages of document development. 
+      </p>
+      <p class="note">
+        See <a href="#bib-smpte-ag-07"></a> for more info regarding document process flowcharts and mapping these status points. 
+      </p>
+      <table id="tab-document-mapping-status" class="col-2-center col-3-center">
+        <caption>Mapping <code>pubState</code> and <code>pubStage</code> to Document Status</caption>
+        <thead>
+          <tr>
+            <th>Document Status</th>
+            <th><code>pubState</code> <code>content</code> value</th>
+            <th><code>pubStage</code> <code>content</code> value</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Undergoing intial draft or revision in DG/SG/WG</td>
+            <td><code>WD</code></td>
+            <td><code>draft</code></td>
+          </tr>
+          <tr>
+            <td>Undergoing pre-FCD review (or) undergoing comment resolution from pre-FCD review (or) undergoing comment resolution from PCD period</td>
+            <td><code>CD</code></td>
+            <td><code>draft</code></td>
+          </tr>
+          <tr>
+            <td>Pre-FCD review complete (or) ready for FCD ballot (or) undergoing FCD ballot (or) ready for PCD (or) currently in PCD period</td>
```

Comment:
```suggestion
            <td>ready for FCD ballot (or) undergoing FCD ballot (or) ready for PCD (or) currently in PCD period</td>
```

"Pre-FCD review complete" does not make it a CD until the comments have been addressed.

Reply:
Addressed in https://github.com/SMPTE/html-pub/pull/139/commits/6ab6d4dd994fa0751b01ac792613709907ee3b96